src/parse/parse_raw.jl

```diff
@@ -329,6 +329,8 @@ function raw_to_exapf(rawdata::Dict{String, Array})
     end
     data["branch"] = branch_array
 
-    return data
+    # TODO:
+    bus_id_to_indexes = Dict{Int, Int}([(i, i) for i in 1:nbus])
```

Comment:
We'll address this after we have the RGM working for a bunch of Matpower examples.

Reply:
Sounds good!